administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -67,7 +67,7 @@ function triggerSave() {
 ?>
 <button id="toogle-left-panel">Show/Hide Reference (PoC)</button>
 
-<form action="<?php echo JRoute::_('index.php?option=com_associations&view=association'); ?>"
+<form action="<?php echo JRoute::_(JFactory::getURI()->toString()); ?>"
```

Comment:
it's better to add the all URL parameters inside JRoute, like all other views.
We need always to be consistent with the rest of the code in the core.